Gemfile

```diff
@@ -41,6 +41,14 @@ gem 'rainbow', '~> 3.0'
 
 gem 'uri_service-client'
 
+gem 'best_type', '0.0.4'
+
+# Fedora 3 related gems
+gem 'rubydora'
+gem 'rest-client'
```

Comment:
I'm not saying we need to change this now if it's complicated to do, but I'd like to recommend using Faraday rather than rest-client.  It's already being pulled in indirectly by the uri_service-client gem, and I believe the samvera community prefers it these days too.

Reply:
rubydora uses rest-client

Reply:
In that case, why are we pulling it in explicitly?

Reply:
TBh I don't remember. Does it run without it?

Reply:
I haven't used rubydora before, so I don't know, but I'm guessing yes because rest-client is included in rubydora's gemspec: https://github.com/samvera/rubydora/blob/master/rubydora.gemspec#L20

Reply:
PR has removed this from the gemfile